File: solutions/2235-capitalize-the-title/solution.py
class Solution(object):
    def capitalizeTitle(self, title):
        """
        :type title: str
        :rtype: str
        """
        title=lower(title)
        ls=title.split()
        for i in range(0,len(ls)):
            if len(ls[i])>2:
                ls[i]=capitalize(ls[i])
        return " ".join(ls)

File: solutions/2235-capitalize-the-title/solution.java
class Solution {
    public String capitalizeTitle(String title) {
        String[] a=title.split("\\ ");
        for(int i=0;i<a.length;i++)
            if(a[i].length()<=2)
                a[i]=a[i].toLowerCase();
            else
                a[i]=a[i].substring(0,1).toUpperCase()+a[i].substring(1).toLowerCase();
        return String.join(" ",a);
    }

}
